resources/views/components/button/primary.blade.php

```diff
@@ -1,3 +1,3 @@
-<x-rapidez::button.base {{ $attributes->class('bg-primary text-primary-text') }}>
+<x-rapidez::button.base {{ $attributes->class('bg-primary text-white') }}>
```

Comment:
Yes, buttons are always custom in projects. there's not much to override either since its only classes you would add to a base.blade.php or index.blade.php